### ISSUE TITLE
Staged recipes not working

### ISSUE THREAD

**Issue by codetaylor** [codetaylor/artisan-worktables#55]
Gamestages: 1.0.78
Artisan: 1.12.21

Script:
```
import mods.artisanworktables.Worktable;
Worktable.addStagedRecipeShaped("ceramics", "mason", <minecraft:gravel>, <ore:masons_trowel>, 1, false, [[<minecraft:clay_ball>, null, <minecraft:clay_ball>], [null, <minecraft:clay_ball>, null]]);
```

**Comment by codetaylor** [comment]
Reference https://github.com/Darkhax-Minecraft/Game-Stages/issues/8

**Comment by codetaylor** [comment]
Confirmed fixed in Game-Stages 1.12.2:1.0.80
